### ISSUE TITLE
Atomic class shoud be reorganized.

### ISSUE THREAD

**Issue by HajimeKawahara** [HajimeKawahara/exojax#513]
I noticed `in moldb.py`, now there are only atomic databases. We should reorganize Adb classes.

**Comment by minouHub** [comment]
Hello,
We have a GSOC working on atomic spectra on the radis side. If you have quick comments to pass before the project is finished, please share them:

- Kurucz database - https://github.com/radis/radis/pull/652
- NIST database - https://github.com/radis/radis/pull/689

**Comment by HajimeKawahara** [comment]
Hello,
Great! Are these implementations included as a common API?　
https://github.com/radis/radis/issues/558
If so, ExoJAX will use these capabilities instead of the internal I/O.

**Comment by minouHub** [comment]
Indeed, the objectives of the GSOC were to implement 2 atomic databases in a new common API. The new databases took more time than expected. Thus, for the moment, NIST and Kurucz will be downloaded as we do for HITRAN, ExoMol, and GEISA... but we are still planing on doing the common API!!!
